internal/debug/flags.go

```diff
@@ -55,6 +55,7 @@ var (
 	memprofilerateFlag = cli.IntFlag{
 		Name:  "memprofilerate",
 		Usage: "Turn on memory profiling with the given rate",
+		Value: runtime.MemProfileRate,
```

Comment:
It's a different story for block profiling though, so its not on by default. But you can turn it on through the API, unlike MemProfileRate.